herddb-core/src/main/java/herddb/model/planner/BindableTableScanOp.java

```diff
@@ -19,54 +19,17 @@
  */
 package herddb.model.planner;
 
-import herddb.core.TableSpaceManager;
-import herddb.model.DataScanner;
-import herddb.model.Projection;
-import herddb.model.ScanResult;
-import herddb.model.StatementEvaluationContext;
-import herddb.model.StatementExecutionException;
-import herddb.model.StatementExecutionResult;
-import herddb.model.TransactionContext;
 import herddb.model.commands.ScanStatement;
-import herddb.utils.Wrapper;
 
 /**
  * TableScanOp + ProjectOp + FilerOp
  *
  * @author eolivelli
  */
-public class BindableTableScanOp implements PlannerOp {
-
-    final ScanStatement statement;
+public class BindableTableScanOp extends SimpleScanOp {
```

Comment:
this is the first of a bunch of semplifications, a bunch of classes had the same implementation, which is not SimpleScanOp